### ISSUE TITLE
firewall rules on VM are not set

### ISSUE THREAD

**Issue by spiderfingers** [Korand/askbot-deployment#1]
**firewalld** on rmq node is run on start and default firewall rules are blocking access to the node via 10.1.1.33:80.

either disable **firewalld** or edit firewall rules.

**Comment by Korand** [comment]
You are wrong. In my puppet manifest:

service { "firewalld":
    ensure => "stopped",
}

**Comment by spiderfingers** [comment]
yeah you got this definition, but check its behavior, in fact it's not being applied. after provision **firewalld** is still up. 
by the way nice work in general anyway =)

**Comment by Korand** [comment]
But during puppet provision I see message: 
==> rmq: Notice: /Stage[main]/Main/Service[firewalld]/ensure: ensure changed 'running' to 'stopped'

And after fresh VM installation I checked firewalld:
 # systemctl status firewalld.service | grep Active
   Active: inactive (dead) since Mon 2015-06-08 21:02:53 UTC; 2min 11s ago

Where is my wrong?

**Comment by spiderfingers** [comment]
i'll look up through that tomorrow and post back. 
and let's agree **task is done**.  my issues were just comments.
please do not waste time on that anymore =) we'll contact you soon
